SlopCrew.sln

```diff
@@ -1,16 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SlopCrew.Plugin", "SlopCrew.Plugin\SlopCrew.Plugin.csproj", "{C33DD7D1-D576-42DC-849A-A67937645689}"
+# Visual Studio Version 17
```

Comment:
All of the project GUIDs seem to have been reset in the solution - unsure what could have caused this but I don't think it'll impact anything.